sdk/python/kfp/_client.py

```diff
@@ -214,18 +220,38 @@ def _get_url_prefix(self):
     # In-cluster pod. We could use relative URL.
     return '/pipeline'
 
-  def create_experiment(self, name, description=None, namespace=''):
+  def set_user_namespace(self, namespace):
+    """Set user namespace into local context setting file.
+       This function should only be used when Kubeflow Pipelines is in the multi-user mode.
+    Args:
+      namespace: kubernetes namespace the user has access to.
+    """
+    self.context_setting['namespace'] = namespace
+    with open(self.LOCAL_KFP_CONTEXT, 'w') as f:
+      json.dump(self.context_setting, f)
+
+  def get_user_namespace(self):
```

Comment:
This method seems to be a little confusing.
It's an a getter method, but it actually changes the client state.

Reply:
That's a valid concern. My thought was to refresh from the file every time we read a value. 
Maybe I should move the file-read part to the class initialization part, so this get can read only from memory.

Reply:
Moved the file read part to a separate method, called during init.